static.config.js

```diff
@@ -118,27 +118,42 @@ export default {
       {
         path: '/',
         component: 'src/js/pages/Home',
+        getProps: () => ({
+          allServicePages: serviceQueries['en'],
+        }),
```

Comment:
This is my own confusion about how react static works, but why are we running serviceQueries for the homepage? I'll be back Monday, and maybe we can pair/talk so I have a better understanding of what's going on. Thanks for being patient, cause I know you've explained this to me before.